telemetry/definitions/commonDefinitions.json

```diff
@@ -142,6 +147,63 @@
             "name": "aws_reportPluginIssue",
             "description": "Report an issue with the plugin"
         },
+        {
+            "name": "cloudwatchlogs_openGroup",
+            "description": "Open the CloudWatch Logs group window",
+            "metadata": [{ "type": "result" }]
+        },
+        {
+            "name": "cloudwatchlogs_openStream",
+            "description": "Open a CloudWatch Logs stream in the window",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_downloadToFile",
+            "unit": "Bytes",
+            "description": "Download a log stream to a file. Value indicates the final size of the formatted log stream.",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_openStreamInEditor",
+            "unit": "Bytes",
+            "description": "Download a log stream to open in editor. Value indicates the final size of the formatted log stream.",
+            "metadata": [{"type": "result"}]
+        },
+        {
+            "name": "cloudwatchlogs_openCurrentInEditor",
```

Comment:
What the difference between this and the above?

Reply:
See the description, download a whole stream and open vs open current view

Reply:
Clarified: both deal with streams in memory. `Open` loads the entire stream (previously unopened) and shows it in the editor pane. `Current messages` shows the messages the user is looking at in the editor pane.